content/en/tracing/service_catalog/_index.md

```diff
@@ -114,6 +114,16 @@ To access additional details describing security vulnerabilities and signals, cl
 
 Click the Settings icon on the right hand corner to hide metric columns from the service list.
 
+### Software Delivery view
+The **Software Delivery tab** provides several ways to assess and improve the pre-production status of your services. This includes understanding the status of your CI pipelines and viewing your static analysis violations. You can:
+
+- View performance of the [CI pipeline(s)][19] related to your services
+- Find security and code quality issues from [Static Analysis][20]
+- Easily pivot from Service Catalog to troubleshoot pre-production slowdowns and failures
+
+To access additional details describing your CI status and static analysis violations, click on a service and see the status of each pipeline and rule violation.
+
+
 ## Investigate a service
 
```

Comment:
Can we add a bullet for Pipeline and Static analysis data under this section? https://a.cl.ly/p9uYL69n

- **Pre-production information** regarding your software delivery process, such as the average build duration and success rate of CI pipelines related to your service, along with static analysis results from CI.